client/delegations.go

```diff
@@ -0,0 +1,213 @@
+package client
+
+import (
+	"github.com/theupdateframework/go-tuf/data"
+	"github.com/theupdateframework/go-tuf/verify"
+)
+
+// getTargetFileMeta searches for a verified TargetFileMeta matching a file name
+// Requires a local snapshot to be loaded and is locked to the snapshot versions.
+// Searches through delegated targets following TUF spec 1.0.19 section 5.6.
+func (c *Client) getTargetFileMeta(file string) (data.TargetFileMeta, error) {
+	snapshot, err := c.loadLocalSnapshot()
+	if err != nil {
+		return data.TargetFileMeta{}, err
+	}
+
+	verifiers := map[string]verify.DelegationsVerifier{
```

Comment:
What are the string keys here? Delegated rolenames?

Reply:
I added comments above to explain the keys https://github.com/theupdateframework/go-tuf/blob/9631aebe2de0c383a399a93f3dcbb0f162d721a9/client/delegations.go#L17-L19
Could you check if they are clear?